example.py

```diff
@@ -26,6 +26,7 @@
         colored=True,                           # If you want to print colored text
         color_palette=ColorPalette(             # You can also create a custom palette color (for the common message).
             STREAMER_online="GREEN",            # Don't worry about lower/upper case the script will be parse all the values.
+            STREAMER_online="GREEN",            # Don't worry about lower/upper case. The script will parse all the values.
```

Comment:
Looks like the line is now duplicated

Reply:
Darn it. I knew I was going to make some kind of an issue. It should be fixed now, sorry :(